src/main/webapp/game.js

```diff
@@ -10,6 +10,10 @@ const CSS_MESSAGE_CLASS_DICT = {
 };
 // TODO: @salilnadkarni, replace with userid from cookie (in datastore)
 const USER_ID = "_" + Math.random().toString(36).substr(2, 9);
+const LOOP_PARAMS = "?version=3&end=10&loop=1&playlist=";
+const EMBED_URL = "https://www.youtube.com/embed/";
+const VIDEO_PARAMS = "&enablejsapi=1&autoplay=1&controls=0&modestbranding=1&disablekb=1&origin=http://localhost:8282";
```

Comment:
I wonder if the "origin=http://localhost:8282" param would cause any issues when the app is deployed and isn't running locally? What is that param used for?

To test it out, can you try deploying your local branch and see if everything works okay (and then re-deploy master afterwards)?

Reply:
I removed and tested on my local server first, and it works without the origin, so I've deleted it. There's an error in the console about it, but it doesn't affect the video playback.

Reply:
@pruess-google